package.json

```diff
@@ -9,33 +9,36 @@
     "url": "http://github.com/codeclimate/codeclimate-eslint.git"
   },
   "dependencies": {
-    "babel-eslint": "^8.2.2",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-airbnb-base": "^12.0.0",
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.7.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-airbnb-base": "^13.1.0",
     "eslint-config-angular": "^0.5.0",
     "eslint-config-dbk": "^3.2.1",
     "eslint-config-drupal": "^4.0.1",
     "eslint-config-ember": "^0.3.0",
     "eslint-config-es5": "^0.5.0",
     "eslint-config-es6": "^1.10.3",
-    "eslint-config-google": "^0.9.1",
+    "eslint-config-google": "^0.11.0",
+    "eslint-config-hapi": "^12.0.0",
     "eslint-config-jquery": "^1.0.1",
-    "eslint-config-loopback": "^8.0.0",
+    "eslint-config-loopback": "^12.0.0",
     "eslint-config-nightmare-mode": "^2.3.0",
     "eslint-config-nodesecurity": "^1.3.1",
     "eslint-config-prettier": "^3.1.0",
-    "eslint-config-secure": "^0.2.1",
+    "eslint-config-react-app": "^3.0.4",
     "eslint-config-signavio": "^3.2.0",
     "eslint-config-signavio-test": "^2.0.0",
-    "eslint-config-standard": "^11.0.0-beta",
-    "eslint-config-standard-jsx": "^4.0.2",
-    "eslint-config-standard-react": "^5.0.0",
+    "eslint-config-simplifield":"^8.1.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-config-standard":"^12.0.0",
```

Comment:
This line is duplicated, also noticed a couple other missing space after `:`, maybe `yarn add eslint-config-secure` would fix all these? (It did for me)